examples/02-http-all.php

```diff
@@ -27,9 +27,7 @@ function ($responses) {
         foreach ($responses as $url => $response) {
             echo $url . ' has ' . $response->getBody()->getSize() . ' bytes' . PHP_EOL;
         }
-    },
-    function ($e) {
-        echo 'An error occurred: ' . $e->getMessage() . PHP_EOL;
+    }, function (Exception $e) {
+        echo 'Error: ' . $e->getMessage() . PHP_EOL;
```

Comment:
Nit: PSR-12 (see above)

```suggestion
    },
    function (Exception $e) {
        echo 'Error: ' . $e->getMessage() . PHP_EOL;
```

Reply:
Answer: changed (see above)